src/entities/Note.ts

```diff
@@ -0,0 +1,45 @@
+import { User } from './User';
+import { IsDate } from 'class-validator';
+import { Entity, Column, ManyToOne, JoinColumn } from 'typeorm';
+
+export interface APINote {
+	ownerID: string;
+	targetUserID: string;
+	noteType: NoteType;
+	time: string;
+}
+
+export enum NoteType {
+	Blocked = 0,
+	Liked = 1
+}
+
+@Entity()
+export default class Note {
```

Comment:
```suggestion
@Unique(['owner', 'targetUser'])
export default class Note {
```

i ran into this yesterday with the Discord entity. Setting this means that we can prevent the same owner making a note against the same user twice.